lib/alchemy/test_support/essence_shared_examples.rb

```diff
@@ -7,15 +7,12 @@
 
   it "touches the content after update" do
     element = create(:alchemy_element)
-    content = create(:alchemy_content, element: element)
+    content = create(:alchemy_content, element: element, updated_at: 3.days.ago)
     essence.save
     content.update(essence: essence, essence_type: essence.class.name)
-    date = content.updated_at
-    Timecop.travel(5.minutes.from_now) do
-      content.essence.update(essence.ingredient_column.to_sym => ingredient_value)
-      content.reload
-      expect(content.updated_at).not_to eq(date)
-    end
+    content.essence.update(essence.ingredient_column.to_sym => ingredient_value)
```

Comment:
Line 12 already changes the value of updated_at, so this test passes even if you comment out line 13. I don't think this was the intent.